gobblin-data-management/src/main/java/org/apache/gobblin/data/management/retention/profile/MultiCleanableDatasetFinder.java

```diff
@@ -52,6 +53,10 @@ public MultiCleanableDatasetFinder(FileSystem fs, Properties jobProps) {
     super(fs, jobProps);
```

Comment:
Should we create a default eventSubmitter here and call MultiCleanableDatasetFinder() which accepts eventSubmitter as an argument?

Reply:
In the super class MultiDatasetFinder, the construction MultiDatasetFinder(fs, jobProps) will create a default eventSubmitter and call MultiDatasetFinder() which accepts eventSubmitter as an argument. So I think there is no need to create a default eventSubmitter here.

Reply:
hmm..ok. Can't we change the super class constructor to MultiDatasetFinder(fs, jobProps, eventSubmitter) and create the submitter in the MultiCleanableDatasetFinder() constructor and pass it to the parent class?